webpack.common.config.js

```diff
@@ -133,7 +133,6 @@ module.exports = Merge.smart({
         plugins: [
             new webpack.ProgressPlugin(), // report progress during compilation
             new webpack.NoEmitOnErrorsPlugin(),
-            new webpack.NamedModulesPlugin(),
```

Comment:
removed based on this comment https://github.com/webpack/webpack/issues/11637#issuecomment-938511019